python/nutpie/compile_pymc.py

```diff
@@ -256,7 +256,7 @@ def _compile_pymc_model_numba(
     for val in [*logp_fn_pt.get_shared(), *expand_fn_pt.get_shared()]:
         if val.name in shared_data and val not in seen:
             raise ValueError(f"Shared variables must have unique names: {val.name}")
-        shared_data[val.name] = val.get_value()
+        shared_data[val.name] = np.array(val.get_value(), order="C", copy=True)
```

Comment:
if you're copying anyway you can do `get_value(borrow=True)` to avoid doing 2 copies